applehat/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "applehat"
```

Comment:
Why Applegate? Just because the LEDs are green by default?

Reply:
Applehat not Applegate. Because Heather likes apples, and the Raspberry Pi it's running on is called applepi.